apps/src/code-studio/pd/pre_workshop_survey/PreWorkshopQuestions.jsx

```diff
@@ -25,16 +25,18 @@ export default class PreWorkshopQuestions extends FormComponent {
             label: `What unit and lesson will you have most recently finished by ${this.props.workshopDate}?`,
             placeholder: "Select a unit",
             required: true,
-            options: units
+            options: units,
+            controlWidth: {md: 6}
```

Comment:
Don't actually do this, but I wish there was a way to not have to specify width for all these components

Reply:
Well, I [made the default](https://github.com/code-dot-org/code-dot-org/pull/17319) full-width `md: 12`. I suppose we could update that to default select controls to `md: 6`, but that might be starting to get complicated. In any case I won't change it for this PR.